core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/MetricType.java

```diff
@@ -89,4 +90,8 @@ public static MetricType fromTextCode(String textCode) {
         return type;
     }
 
+    public static List<MetricType> userTypes() {
+        return Arrays.asList(Arrays.copyOf(MetricType.values(), 3));
```

Comment:
How about creating a static final `EnumSet` instead of creating a list on each call?